lighthouse-core/lib/styles-helpers.js

```diff
@@ -21,25 +21,34 @@
  * or name/value pair.
  *
  * @param {!Array} stylesheets A list of stylesheets used by the page.
- * @param {string=} propName Optional name of the CSS property to filter results
- *     on. If propVal is not specified, all stylesheets that use the property are
+ * @param {string|Array<string>=} propName Optional name of the CSS property/propertys to filter
+ *     results on. If propVal is not specified, all stylesheets that use the property are
  *     returned. Otherwise, stylesheets that use the propName: propVal are returned.
- * @param {string=} propVal Optional value of the CSS property to filter results on.
+ * @param {string|Array<string>=} propVal Optional value of the CSS property/propertys to filter
+ *     results on.
  * @return {!Array} A list of stylesheets that use the CSS property.
  */
 function filterStylesheetsByUsage(stylesheets, propName, propVal) {
   if (!propName && !propVal) {
     return [];
   }
-
   // Create deep clone of arrays so multiple calls to filterStylesheetsByUsage
   // don't alter the original artifacts in stylesheets arg.
   const deepClone = stylesheets.map(sheet => Object.assign({}, sheet));
 
   return deepClone.filter(s => {
     s.parsedContent = s.parsedContent.filter(item => {
-      const usedName = item.property.name === propName;
-      const usedVal = item.property.val.indexOf(propVal) === 0; // val should start with needle
+      let usedName = '';
+      let usedVal = '';
+      // Prevent indexOf on null value
+      if (propName) {
+        propName = Array.from(propName);
```

Comment:
These aren't quite what you want. If the params are strings, you'll get something like:

`Array.from('display')` -> `["d", "i", "s", "p", "l", "a", "y"]`.

these will work:
```
Array.isArray(propName) ? propName : [propName]
Array.isArray(propVal) ? propVal : [propVal]
```